Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.Properties.cs

```diff
@@ -15,6 +15,26 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// <seealso cref="Windows.UI.Xaml.Controls.ItemsControl" />
     public partial class MasterDetailsView
     {
+        /// <summary>
```

Comment:
It looks like properties have been moved around. This is making it really difficult to see what actually changed. Can you put the properties back in the order they were originally.

Reply:
OK. I will work on it tomorrow.
I've rearranged them to make them a little more logical.

Reply:
That could be done as a separate "no functional change" PR

Reply:
Ok. I've rearranged them.